package.json

```diff
@@ -61,11 +61,11 @@
     "debug": "2.6.9",
     "express": "4.16.2",
     "express-force-ssl": "0.3.2",
-    "express-graphql": "0.6.11",
+    "express-graphql": "0.9.0",
     "express-ipfilter": "0.3.1",
     "express-rate-limit": "3.2.0",
     "graphiql": "0.11.11",
-    "graphql": "0.13.2",
+    "graphql": "14.5.4",
```

Comment:
WHAT?! am i reading it right?! we were 14 major versions behind?!

Reply:
😄 Similar to this https://reactjs.org/blog/2016/02/19/new-versioning-scheme.html